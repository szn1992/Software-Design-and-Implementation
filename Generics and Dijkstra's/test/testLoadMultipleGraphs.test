# Test if a graph can be loaded with multiple file

# Load multiple files to the same graph
LoadGraph graph marvel.tsv
LoadGraph graph staffSuperheroes.tsv
LoadGraph graph marvel.tsv
LoadGraph graph staffSuperheroes.tsv

# list children of one node
ListChildren graph Ernst-the-Bicycling-Wizard