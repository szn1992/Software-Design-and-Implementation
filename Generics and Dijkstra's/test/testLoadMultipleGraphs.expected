# Test if a graph can be loaded with multiple file

# Load multiple files to the same graph
loaded graph graph
loaded graph graph
loaded graph graph
loaded graph graph

# list children of one node
the children of Ernst-the-Bicycling-Wizard in graph are: Notkin-of-the-Superhuman-Beard(0.500) Perkins-the-Magical-Singing-Instructor(1.000)